src/main/resources/templates/repo/ebextensions/beanstalk_logs_cloudwatch.config

```diff
@@ -0,0 +1,19 @@
+#### See http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/AWSHowTo.cloudwatchlogs.html
+packages:
+  yum:
+    awslogs: []
+
+files:
```

Comment:
Also try to add the repository-services.log and repo-trace-profile.log. See https://github.com/Sage-Bionetworks/Synapse-Repository-Services/blob/develop/services/repository/src/main/resources/log4j2.xml.